conjure-object/src/safe_long.rs

```diff
@@ -132,3 +147,24 @@ impl fmt::Display for BoundsError {
 }
 
 impl Error for BoundsError {}
+
+#[derive(Debug, Clone)]
+enum ParseErrorInner {
+    Parse(ParseIntError),
+    Bounds(BoundsError),
+}
+
+/// The error returned after failing to parse a string into a `SafeLong`.
+#[derive(Debug, Clone)]
+pub struct ParseError(ParseErrorInner);
+
+impl fmt::Display for ParseError {
```

Comment:
Do you have thoughts on [enum-display-derive](https://github.com/ihrwein/enum-display-derive)? I'm fine with this explicit implementation but I'm curious if there is an explicit reason not to use this crate.

Reply:
Doesn't seem worth it to pull in an extra dependency for one type.